pkg/api/types.go

```diff
@@ -72,10 +72,11 @@ type ExportSpec struct {
 	// have a different `Name`. The endpoint Host name could be different from Name,
 	// decoupling service names between Peers.
 	Service Endpoint
-	// ExternalService endpoint defines an endpoint located outside of the cluster.
-	// When this endpoint is defined, a new local Endpoint will be created.
-	// This local Endpoint serves as a bridge, directing the Service to the ExternalService.
-	ExternalService Endpoint
+	// ExternalService specifies a service located outside of the cluster.
+	// If ExternalService is defined, a new local ExternalName Service will be created,
+	// pointing to the ExternalService. Note that the port value is the same as defined
+	// in the Service field.
```

Comment:
Note: the external name is **just a DNS name mapping** (i.e., when some workload resolves "servicename.namespace.cluster.local" CoreDNS return the IP address associated with "externalname").
Unlike regular services, there is **no port mapping** involved and no Service VIP allocated.

Reply:
Check now if it is more clear.